envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoycontrol/EnvoyControlTestApp.kt

```diff
@@ -228,7 +228,7 @@ class EnvoyControlRunnerTestApp(
 
 data class Health(
     val status: Status,
-    val details: Map<String, HealthDetails>
+    val components: Map<String, HealthDetails>
```

Comment:
This rename seems like an unrelated change. Can you explain?

Reply:
Spring changed the format of the health endpoint somewhere between 2.1 and 2.3 and details is gone now.